momento-sdk/src/intTest/java/momento/sdk/AuthClientTopicTests.java

```diff
@@ -0,0 +1,515 @@
+package momento.sdk;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import momento.sdk.auth.accessControl.CacheSelector;
+import momento.sdk.auth.accessControl.DisposableTokenScopes;
+import momento.sdk.auth.accessControl.ExpiresIn;
+import momento.sdk.auth.accessControl.TopicSelector;
+import momento.sdk.exceptions.MomentoErrorCode;
+import momento.sdk.responses.auth.GenerateDisposableTokenResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AuthClientTopicTests extends BaseTestClass {
```

Comment:
super nit but I think this should be `AuthClientTests`?

Reply:
The tests are split into AuthClientCacheTests and AuthClientTopicTests. Hence, the naming!

Reply:
ah ok I see its separated by permissions